pkg/transfers/inbound/cleanup.go

```diff
@@ -53,3 +59,13 @@ func deleteFilesOnRemote(logger log.Logger, agent upload.Agent, localDir, suffix
 	}
 	return el
 }
+
+// ShouldDeleteEmptyFile determines if a file is empty and if it should be deleted
+// per the config setting RemoveEmptyFileAfter
+func ShouldDeleteEmptyFile(info os.FileInfo, removeEmptyFileAfter int) bool {
+	timeNow := time.Now()
+	fileCreationTime := info.ModTime()
+	diff := fileCreationTime.Sub(timeNow)
+	timeOnServer := diff.Minutes()
+	return info.Size() == 0 && int(timeOnServer) >= removeEmptyFileAfter
```

Comment:
A suggestion for simplifying the code a bit:
```go
if info.Size() != 0 {
 return false
}
diff := time.Now().Sub(info.ModTime())
return int(diff.Minutes) >= removeEmptyFileAFter
```

Reply:
made it private